sound/soc/sof/ipc.c

```diff
@@ -207,6 +207,7 @@ static int tx_wait_done(struct snd_sof_ipc *ipc, struct snd_sof_ipc_msg *msg,
 		dev_err(sdev->dev, "error: ipc timed out for 0x%x size %d\n",
 			hdr->cmd, hdr->size);
 		snd_sof_dsp_dbg_dump(ipc->sdev, SOF_DBG_REGS | SOF_DBG_MBOX);
+		snd_sof_dsp_ipc_regs_dump(ipc->sdev);
```

Comment:
snd_sof_ipc_dump? we have IPC registers but others may have more stuff to log?

Reply:
I also thinks about `snd_sof_dsp_dbg_dump(ipc->sdev, SOF_DBG_REGS | SOF_DBG_MBOX | SOF_DBG_IPC);`
but I found that we never used the `flag `in `snd_sof_dsp_dbg_dump(struct snd_sof_dev *sdev, u32 flags)`
So I just create a new function for this.